src/basic/missing_audit.h

```diff
@@ -4,21 +4,29 @@
 #include <linux/audit.h>
 
 #if HAVE_AUDIT
-#include <libaudit.h>
+#  include <libaudit.h>
 #endif
 
 #ifndef AUDIT_SERVICE_START
-#define AUDIT_SERVICE_START 1130 /* Service (daemon) start */
+#  define AUDIT_SERVICE_START 1130 /* Service (daemon) start */
+#else
+static_assert(AUDIT_SERVICE_START == 1130);
```

Comment:
Maybe, better to unconditionally define a hidden value, then use that?
```c
#define _SD_AUDIT_SERVICE_START 1130
#ifndef AUDIT_SERVICE_START
#  define AUDIT_SERVICE_START _SD_AUDIT_SERVICE_START
#else
static_assert(AUDIT_SERVICE_START == _SD_AUDIT_SERVICE_START);
#endif
```

Reply:
BTW, why `static_assert()`? We usually use `assert_cc()`, no?

Reply:
I always get them confused. I'll change to `assert_cc`.

Reply:
> Maybe, better to unconditionally define a hidden value, then use that?
> 
> ```c
> #define _SD_AUDIT_SERVICE_START 1130
> #ifndef AUDIT_SERVICE_START
> #  define AUDIT_SERVICE_START _SD_AUDIT_SERVICE_START
> #else
> static_assert(AUDIT_SERVICE_START == _SD_AUDIT_SERVICE_START);
> #endif
> ```

I forgot to answer here: I thought about this, but it'd make the defines even longer. So I think it's OK to repeat the numeric constant twice, but save one line of text. We generally only write those lines once and never edit them again, so I don't think the duplication will cause any problems.